docs/settings.md

```diff
@@ -54,6 +55,13 @@ Setting the TOKEN_TTL to `None` will create tokens that never expire.
 Warning: setting a 0 or negative timedelta will create tokens that instantly expire,
 the system will not prevent you setting this.
 
+## TOKEN_LIMIT
```

Comment:
I'd prefer a name that does not imply there is a general token limit but emphasizes a per-user token limit.

Reply:
Like `TOKEN_LIMIT_PER_USER`? I will make sure to fix this once this has been decided https://github.com/James1345/django-rest-knox/pull/114.